runtime/python/pose_estimation/pose_estimation_utils.py

```diff
@@ -0,0 +1,506 @@
+from pathlib import Path
+import numpy as np
+import cv2
+from PIL import Image
+from hailo_platform import HEF
+from zenlog import logging
+
+# Joint pairs used for drawing pose estimations
+JOINT_PAIRS = [
+    [0, 1], [1, 3], [0, 2], [2, 4],
+    [5, 6], [5, 7], [7, 9], [6, 8], [8, 10],
+    [5, 11], [6, 12], [11, 12],
+    [11, 13], [12, 14], [13, 15], [14, 16]
+]
+
+
+def data_type2dict(net: str, data_type: str) -> dict:
```

Comment:
name is not very clear, maybe output_type2dict?

Reply:
output_data_type2dict